test/incompatibilities.js

```diff
@@ -85,6 +85,13 @@ describe("Parser", function() {
       )
     );
 
+    // ES5: does not allow duplicate property names in strict mode
+    // ES6: allows duplicate property names except for __proto__
+    expect(stmt(parse("function hello() {\'use strict\'; ({ i: 42, i: 42 }) }",
```

Comment:
This is not the signature of `expect`. These error messages shouldn't even be here.